awx/main/tasks.py

```diff
@@ -905,6 +905,46 @@ def build_params_process_isolation(self, instance, private_data_dir, cwd):
                 process_isolation_params['process_isolation_ro_paths'].append(instance.ansible_virtualenv_path)
         return process_isolation_params
 
+    def build_params_resource_profiling(self, instance, private_data_dir):
+        resource_profiling_params = {}
+        if self.should_use_resource_profiling(instance):
+            base_cgroup = getattr(settings, 'AWX_RESOURCE_PROFILING_BASE_CGROUP', 'ansible-runner')
+            cpu_poll_interval = getattr(settings, 'AWX_RESOURCE_PROFILING_CPU_POLL_INTERVAL', '0.25')
+            mem_poll_interval = getattr(settings, 'AWX_RESOURCE_PROFILING_MEMORY_POLL_INTERVAL', '0.25')
+            pid_poll_interval = getattr(settings, 'AWX_RESOURCE_PROFILING_PID_POLL_INTERVAL', '0.25')
+
+            results_dir = os.path.join(private_data_dir, 'artifacts/playbook_profiling')
+            if not os.path.isdir(results_dir):
+                os.makedirs(results_dir, stat.S_IREAD | stat.S_IWRITE | stat.S_IEXEC)
+
+            logger.debug('Collected the following resource profiling intervals: cpu: {} mem: {} pid: {}'
+                         .format(cpu_poll_interval, mem_poll_interval, pid_poll_interval))
+
+            if not isinstance(cpu_poll_interval, str):
```

Comment:
Since these end up needing to be coerced to `str` anyways, is there actually any value in storing these using `FloatField` (would it simplify this PR to just use `CharField`?)

Reply:
Actually, do we need _this_ `try...except...` code at all?  It seems like we're already coercing these values here:

https://github.com/ansible/ansible-runner/blob/master/ansible_runner/runner_config.py#L221

Reply:
absolutely - not needed anymore, removed